### PR TITLE
Update Get-SPOCrossGeoMovedUsers.md

### PR DESCRIPTION
**Pull request by PowershellScripts**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossGeoMovedUsers.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Get-SPOCrossGeoMovedUsers -Direction <String> [<CommonParameters>]
 
 ## DESCRIPTION
 
-This cmdlet allows you to get the moved users out and in the current SPO Site. It requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global Administrator to get the moved users out and in the current SPO site.
+This cmdlet allows you to get the moved users out and in the current SPO Site. It requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online administrator or Global Administrator to get the moved users out and in the current SPO site.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossGeoMovedUsers.md:27]:
```suggestion
This cmdlet allows you to get the moved users out and in the current SPO Site. It requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online Administrator or Global Administrator to get the moved users out and in the current SPO site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@PowershellScripts

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**